test/topology_experimental_raft/test_tablets.py

```diff
@@ -753,3 +757,75 @@ async def test_tablet_count_metric_per_shard(manager: ManagerClient):
     # And then tablet count metric is increased on dest_host - tablets have been moved to shard_3
     dest_expected_count_per_shard[3] += count_of_tokens_on_src_shard_to_move
     await assert_tablet_count_metric_value_for_shards(manager, dest_server, dest_expected_count_per_shard)
+
+async def test_tablet_load_and_stream(manager: ManagerClient):
+    logger.info("Bootstrapping cluster")
+    cmdline = [
+        '--logger-log-level', 'storage_service=debug',
+        '--logger-log-level', 'table=debug',
+        '--smp', '1',
+    ]
+    servers = [await manager.server_add(cmdline=cmdline)]
+
+    await manager.api.disable_tablet_balancing(servers[0].ip_addr)
+
+    cql = manager.get_cql()
+    # Creates multiple tablets in the same shard
+    await cql.run_async("CREATE KEYSPACE test WITH replication = {'class': 'NetworkTopologyStrategy', 'replication_factor': 1}" \
+                        " AND tablets = {'initial': 5};") # 5 is rounded up to next power-of-two
```

Comment:
Why not use 8 directly? Are you testing the round up?

Reply:
Kind of, but it's irrelevant to the test itself. But it doesn't hurt.